.github/workflows/scheduled.yml

```
@@ -0,0 +1,60 @@
+name: Scheduled jobs
+
+on:
+  schedule:
+   - cron: "15 5 * * 1"
+
+jobs:
+  check-broken-links:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.DEPLOYMENT_PERSONAL_ACCESS_TOKEN }}
+          submodules: true
+
+      - name: Check for broken links
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.5.4
+        with:
+          args: --verbose ./docs/**/*.md ./submodules/chart/docs/**/*.md
+          jobSummary: true
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+      - name: Create issue
+        if: steps.lychee.outputs.exit_code != 0
+        uses: peter-evans/create-issue-from-file@v4
+        with:
+          title: Broken link report
+          content-filepath: ./lychee/out.md
+          labels: report, automated issue
+
+  compress-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.DEPLOYMENT_PERSONAL_ACCESS_TOKEN }}
+          submodules: true
+
+      - name: Compress images
+        id: calibre
+        uses: calibreapp/image-actions@main
+        with:
+          githubToken: ${{ secrets.DEPLOYMENT_PERSONAL_ACCESS_TOKEN }}
+          ignorePaths: submodules/**
+          compressOnly: true
```

I believe we can remove this one too

```suggestion
          githubToken: ${{ secrets.DEPLOYMENT_PERSONAL_ACCESS_TOKEN }}
          compressOnly: true
```